src/makeCalculator.js

```diff
@@ -37,7 +37,39 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+
+    add: (a, b) => {
+      return a + b;
+    },
```

Comment:
We can write shorthand syntax here
```suggestion
    add: (a, b) => (a + b),
```